java-extras/src/main/java/org/triplea/io/FileUtils.java

```diff
@@ -3,13 +3,26 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 
 import java.io.File;
+import java.util.ArrayList;
 import java.util.Collection;
 import java.util.List;
 import java.util.Optional;
+import lombok.experimental.UtilityClass;
 
 /** A collection of useful methods related to files. */
+@UtilityClass
 public final class FileUtils {
-  private FileUtils() {}
+
+  /**
+   * Creates a new file with a parent folder and any number of child folders. This is a convenience
+   * method to concatenate the path together with an OS specific file separator.
+   */
+  public static File newFile(final String parentDir, final String... childDirs) {
+    final List<String> dirs = new ArrayList<>();
+    dirs.add(parentDir);
+    dirs.addAll(List.of(childDirs));
+    return new File(String.join(File.separator, dirs));
```

Comment:
Thought: I we would use `java.nio.file.Path` instead of `java.io.File` we could easily use [`Path#of`](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Path.html#of(java.lang.String,java.lang.String...)) for convinience instead of coding this logic ourselves.

Reply:
That being said, you could simplify this code to
```java
return Path.of(parentDir, chilDirs).toFile();
```